build.sbt

```diff
@@ -345,4 +346,19 @@ lazy val `mesos-client` = (project in file("mesos-client"))
     PB.targets in Compile := Seq(
       scalapb.gen() -> (sourceManaged in Compile).value
     )
-  )
\ No newline at end of file
+  )
+
+lazy val chaos = (project in file("chaos"))
+  .configs(IntegrationTest)
+  .settings(commonSettings: _*)
+  .settings(formatSettings: _*)
+  .settings(
+    libraryDependencies ++= Dependencies.chaos)
+  .dependsOn(`chaos-jersey-guice-14`)
+
+lazy val `chaos-jersey-guice-14` = (project in file("chaos/jersey-guice"))
```

Comment:
What is the idea of this project?

Reply:
This is brought over from Chaos. I'm in the process of removing it. Baby steps.